app/src/main/java/org/fossasia/openevent/app/common/data/models/Event.java

```diff
@@ -32,10 +44,18 @@
 @AllArgsConstructor
 @JsonNaming(PropertyNamingStrategy.KebabCaseStrategy.class)
 @Table(database = OrgaDatabase.class, allFields = true)
+@JsonInclude(JsonInclude.Include.NON_DEFAULT)
 @EqualsAndHashCode(exclude = { "eventDelegate", "analytics" })
 @SuppressWarnings({ "PMD.ExcessivePublicCount", "PMD.TooManyFields" })
+@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY, getterVisibility = JsonAutoDetect.Visibility.NONE,
```

Comment:
Why is this needed?

Reply:
There are some fields like `isMapShown` which by default are getting serialized to two fields - `is-map-shown` and `map-shown`. So to prevent this I added the above annotation after which it as only getting serialized to `is-map-shown`.

Reply:
Yes